src/main/resources/application-local.yml

```diff
@@ -16,3 +16,4 @@ logging:
 spring:
   flyway:
     locations: classpath:db/migration,classpath:db/local
+    clean-on-validation-error: true
```

Comment:
I'm assuming this "clean if migrations are invalid" is back because it's useful in the local env, especially when it comes to seed data changing.

_Not part of this PR_
Would it make sense to create a standalone command for seeding, not treating it as a migration?

For example, in Rails and Phoenix there's a `bin/rake db:seed` and `mix run priv/repo/seeds.exs` for similar purposes, and it's "on demand" rather than automatic. (And then people usually have a shortcut "reset" commands that drop/create/migrate/seed.)

Reply:
_Edit_
Seems like repeatable migrations might have similar semantics (emphasis mine):

> Repeatable migrations have a description and a checksum, but no version. Instead of being run just once, they are (re-)applied **every time their checksum changes**

This might work well for local data.

Reply:
ah no - this is a mistake - it should not be back! removing now. good spot.